presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -37,6 +47,10 @@ for your Hive metastore Thrift service:
     connector.name=hive-cdh4
     hive.metastore.uri=thrift://example.net:9083
 
+Additionally, you should add the following property to ``jvm.config``, replacing <hdfs_username> with your hdfs user name: ::
```

Comment:
I read the code and I understand it now. This should go in the HDFS section below.